estate_account/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+{
+    'name': "18.0-estate-x-invoicing-umbm",
+    'summary': "Short (1 phrase/line) summary of the module's purpose",
+    'author': "My Company",
+    'website': "https://www.yourcompany.com",
+    'license': 'GPL-3',
+    'category': 'Uncategorized',
+    'version': '0.1',
+    'depends': ['base', 'account', 'estate'],
```

Comment:
```suggestion
    'depends': ['account', 'estate'],
```
it's ok to omit `'base'` here because it's a dependency of the others